include/tester/vtcp/tcp/TCPServer.hpp

```diff
@@ -3,22 +3,23 @@
 #include<iostream>
 #include<memory>
 #include<unordered_map>
+#include<type_traits>
 
 #include<boost/asio.hpp>
 
 #include"TCPSession.hpp"
 #include"TCPListener.hpp"
-#include"../generic/RequestHandler.hpp"
 
 using boost::asio::ip::tcp;
 
+template <class T>
```

Comment:
change `class` to `typename` in the template declaration

Reply:
done